protobuf/api/execution.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	fmt "fmt"
+
+	execution "github.com/mesg-foundation/engine/execution"
+	"github.com/mesg-foundation/engine/x/xstrings"
+)
+
+// Validate checks if given filter is valid and returns error.
+func (f *StreamExecutionRequest_Filter) Validate() error {
+	if f == nil {
+		return nil
+	}
+
+	if !f.ExecutorHash.Valid() {
+		return fmt.Errorf("stream filter: executor hash is invalid")
+	}
+
+	if !f.InstanceHash.Valid() {
+		return fmt.Errorf("stream filter: instance hash is invalid")
+	}
+
+	// TODO: add validation (after adding in protobuf with print ascii)
```

Comment:
See comment https://github.com/mesg-foundation/engine/pull/1463#discussion_r348413155

Reply:
This is a validation about TaskKey (with go validator)